plugins/it.xsemantics.dsl/src/it/xsemantics/dsl/XsemanticsModelGenerator.xtend

```diff
@@ -0,0 +1,68 @@
+/*******************************************************************************
```

Comment:
As @mor-n4 said, this modifies the feature that @JensN4 implemented a while ago: Before this modification here, Jens' feature used common comment syntax for given copyright headers in the .xsemantics file. With the modification, the comment syntax must be given in the .xsemantics file explicitly. Please see my comment (https://github.com/LorenzoBettini/xsemantics/pull/95#discussion_r127663769) for an example use case of the modified copyright feature.